package.json

```diff
@@ -107,11 +107,12 @@
   "devDependencies": {
     "@apollo/react-testing": "^4.0.0",
     "@babel/core": "^7.7.0",
-    "@testing-library/user-event": "^13.2.1",
     "@testing-library/jest-dom": "^5.3.0",
     "@testing-library/react": "^10.0.2",
     "@testing-library/react-hooks": "^3.4.2",
+    "@testing-library/user-event": "^13.2.1",
     "@theforeman/builder": "^15.0.0",
+    "@theforeman/eslint-plugin-rules": "^15.0.1",
```

Comment:
Please do not add imports from `@theforeman/*` library 
this now lives in: https://github.com/theforeman/foreman/tree/develop/script/lint/%40theforeman/eslint-plugin-rules

Reply:
Yeah, sorry, this file got here by mistake. Deleting.